src/data/nat/prime.lean

```diff
@@ -645,6 +645,19 @@ have hn : 0 < n := nat.pos_of_ne_zero $ λ h, begin
 end,
 perm_of_prod_eq_prod (by rwa prod_factors hn) h₂ (@prime_of_mem_factors _)
 
+lemma prime_pow_factors (p y : ℕ) [hp : fact p.prime] :
+  (p ^ y).factors = list.repeat p y :=
+begin
+  symmetry,
+  rw ← list.repeat_perm,
+  apply nat.factors_unique,
+  { apply list.prod_repeat },
+  { intros q hq,
+  rw (list.mem_repeat.1 hq).right,
+  exact fact_iff.symm.2 hp,
+  },
```

Comment:
```suggestion
    rw (list.mem_repeat.1 hq).right,
    exact fact_iff.symm.2 hp },
```
Indentation in the bracket, and mathlib style is for the closing bracket to be on the same line.